code/modules/client/preference/link_processing.dm

```diff
@@ -1070,6 +1070,125 @@
 					INVOKE_ASYNC(user.client, /client.proc/edit_2fa)
 					return // We return here to avoid focus being lost
 
+				if("keybindings")
+					if(!keybindings_overrides)
+						keybindings_overrides = list()
+
+					if(href_list["set"])
+						var/datum/keybinding/KB = locateUID(href_list["set"])
+						if(KB)
+							if(href_list["key"])
+								var/old_key = href_list["old"]
+								var/new_key = copytext(url_decode(href_list["key"]), 1, 16)
+								var/alt_mod = text2num(href_list["alt"]) ? "Alt" : ""
+								var/ctrl_mod = text2num(href_list["ctrl"]) ? "Ctrl" : ""
+								var/shift_mod = text2num(href_list["shift"]) ? "Shift" : ""
+								var/numpad = (text2num(href_list["numpad"]) && text2num(new_key)) ? "Numpad" : ""
+								var/clear = text2num(href_list["clear_key"])
+
+								if(new_key == "Unidentified") //There doesn't seem to be any any key!
+									capture_keybinding(user, KB, href_list["old"])
```

Comment:
Why are we capturing undefined if it isn't a key?

Reply:
Oh github was relining in a scuffed way, I didnt see `old` as the arg.

Reply:
Yah, this just puts it back to whatever it was before and nopes out